packages/agent/conf/demos/fmdm/oid4vci_metadata/fmdm.json

```diff
@@ -59,40 +59,40 @@
           }
         ],
         "order": [
-          "Voornaam",
-          "Achternaam",
+          "firstName",
+          "lastName",
           "email",
           "type"
         ],
         "credentialSubject": {
-          "Voornaam": {
+          "firstName": {
             "value_type": "string",
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
               },
               {
-                "name": "FirstName",
+                "name": "First Name",
                 "locale": "en-US"
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
                 "locale": "nl-NL"
               }
             ]
           },
-          "Achternaam": {
+          "lastName": {
             "value_type": "string",
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```

Comment:
what should be the default locale here? dutch or english?

Reply:
see prev answer

Reply:
we support english and dutch for now, but i believe some parties want dutch as default and others english